app/scripts/helpers/price.helpers.js

```diff
@@ -0,0 +1,3 @@
+export function formatPrice(amount, currency) {
+	return `${currency === 'USD' ? '$' : ''}${amount}${currency === 'EUR' ? ' €' : ''}`;
```

Comment:
Not sure if its necessary but there is a native way to produce a currency according to ISO spec. https://medium.com/@daveford/format-currency-in-javascript-2b8721dc67c7#.ggby7u9yl